src/common/AdaptiveCencSampleDecrypter.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Copyright (C) 2023 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
```

Comment:
this and the other one added to wvdecrypter*.cpp
looks like a workarounds maybe due to circular includes